tock/api/tests.py

```diff
@@ -39,11 +46,17 @@ def test_projects_csv(self):
 class ProjectInstanceAPITests(WebTest):
     fixtures = FIXTURES
 
+    def setUp(self):
+        self.user = User.objects.get(username='aaron.snow')
+        self.token = Token.objects.get_or_create(user=self.user)[0].key
+        self.client = APIClient()
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+
```

Comment:
addressed via https://github.com/18F/tock/pull/465/commits/462a01c564e669a9375ed298f7279e3bbb441ac7 and https://github.com/18F/tock/pull/465/commits/c4b53fbba479ba3a3299cc008bec22eabaa68c0d